tests/test_client.py

```diff
@@ -251,6 +251,36 @@ def test_no_user_raise(self, client):
         with pytest.raises(DeezerErrorResponse):
             client.get_user(-1)
 
+    def test_get_user_recommendations_track(self, client):
```

Comment:
Forgot to mention that for all the tests that need an API token, you should replace the `client` fixture by this one:

https://github.com/browniebroke/deezer-python/blob/main/conftest.py#L20-L23

Reply:
Ohhh, ok. I was getting a little confused why the tests kept failing🙃 Im sorry for this mess of a PR btw

Reply:
> Im sorry for this mess of a PR btw

Don't worry about it, I'll squash the commits when merging, it'll show up nicely in the main branch.